app/code/core/Mage/Wishlist/controllers/IndexController.php

```diff
@@ -637,7 +637,7 @@ public function sendAction()
             return $this->norouteAction();
         }
 
-        $emails  = explode(',', $this->getRequest()->getPost('emails'));
+        $emails  = array_filter(explode(',', $this->getRequest()->getPost('emails')));
```

Comment:
Unless the second param is provided, `getPost()` can return `null`, but `explode()` expects its second param to be a string. I suggest:
```suggestion
        $emails  = array_filter(explode(',', $this->getRequest()->getPost('emails', '')));
```